velocidi-sdk/src/main/kotlin/com/velocidi/events/CustomTrackingEvent.kt

```diff
@@ -11,10 +11,14 @@ class CustomTrackingEvent(
     val extraAttributes: JSONObject = JSONObject()
```

Comment:
I know this is not from this PR, but if we don't have a concrete use case, I would prefer not to include the functionality to get a `CustomTrackingEvent` from JSON. Also, I think the exceptions thrown should be custom (like if a given field cannot be extracted from the input JSON) and are, at this moment, generic JSON exceptions.

Reply:
I think we can iterate on this later. I'll create an issue with this.

Reply:
https://github.com/velocidi/velocidi-android-sdk/issues/8